lldp/lldp.yaml

```diff
@@ -0,0 +1,37 @@
+components:
+  schemas:
+    Lldp:
+      description: >-
+        Configuration of LLDP protocol
+        IEEE Ref: https://www.ieee802.org/1/files/public/docs2002/lldp-protocol-00.pdf
+      type: object
+      required: [connection]
+      properties:
+        connection:
+          description: >-
+            The unique name of the object on which LLDP is running. 
+          $ref: '#/components/schemas/Lldp.Connection'
+          x-field-uid: 1
+        config:
```

Comment:
config can include other items such as holdtime etc. which is related to operation of LLDP .   Information needed to create TLVs + operational config such as timers are aggregated in the config. So I dont think this can become tlvs .

Reply:
I think the idea was to clearly separate the connection info ( mandatory ) to clearly indicate the per port LLDP control at top level and bunch all other parameters into a separate optional container ( config) which could potentially be re-used as a x-include able object for LLDP ( over emulated interfaces for example, in future which would have 'config' + emulated interface name if kept inside device )

Reply:
pls check https://github.com/openconfig/public/blob/master/release/models/lldp/openconfig-lldp.yang, different config elements are in the same object in our case while they have in different object. We have in one as we wanted to keep in simple and the config elements are very few,but it can get extended in future.Instead of config shall we use attributes/parameters?